server/api/users.js

```diff
@@ -12,3 +18,36 @@ router.get('/', (req, res, next) => {
     .then(users => res.json(users))
     .catch(next)
 })
+
+// GET /api/users/:userId
+router.get('/:userId', (req, res, next) => {
+    const id = req.params.userId;
+    User.findOne({
+        where: {id},
+        attributes: ['id', 'email']
+      })
+    .then(user => res.json(user))
+    .catch(next);
+})
+
+// PUT /api/users/:userId/update
+router.put('/:userId/update', (req, res, next) => {
```

Comment:
I would just User.update instead of User.findOne(...).then(user => user.update(...))